util/logtools/make_timeline.ts

```diff
@@ -512,6 +528,54 @@ const assembleTimelineStrings = (
     }
     lastEntry = entry;
   }
+  const headerLines = assembleHeaderAbilityStrings(args, encounterAbilityList);
+  return headerLines.concat(assembled);
+};
+
+const assembleHeaderZoneInfoStrings = (fight: FightEncInfo): string[] => {
+  const zoneName = fight.zoneName;
+  const zoneId = fight.zoneId;
+  const headerInfo = [];
+  if (zoneName !== undefined) {
+    const zoneNameLine = `### ${zoneName.toUpperCase()}`;
+    headerInfo.push(zoneNameLine);
+  }
+  if (zoneId !== undefined) {
+    const zoneIdLine = `# ZoneID: ${zoneId}`;
```

Comment:
```suggestion
    const zoneIdLine = `# ZoneId: ${zoneId}`;
```

The tiniest of nitpicks, but maybe `ZoneId` to match `ZoneId.SomeZoneTitleHere` in code.